libs/networking/http_mock_client/Http_mock_client.ml

```diff
@@ -105,12 +105,13 @@ let body_of_file ?(trim = false) path =
 let check_body expected_body actual_body =
   let%lwt actual_body_content = Cohttp_lwt.Body.to_string actual_body in
   let%lwt expected_body_content = Cohttp_lwt.Body.to_string expected_body in
-  Alcotest.(check string) "body" expected_body_content actual_body_content;
+  (* Using "" prevents always printing "ASSERT name" to stderr *)
```

Comment:
If we pass an empty string it just doesn't print anything?

Reply:
I don't understand andre's comment

Reply:
Currently we print `ASSERT body` etc... to stderr when the mock is used. For the most part this is captured but when there's a failing test we can get a decent amount of printout since it'll then show stderr.

Reply:
I will tidy up the comment since it doesn't make sense for those without context, but yes I was looking at the code of the library an noticed that the print statement checks if the string is empty first.